main.go

```diff
@@ -84,7 +85,7 @@ func (d *DruidTasksExporter) Collect(ch chan<- prometheus.Metric) {
 				}
 			}
 			if !is_present {
-				tasks = append(tasks, Task{Type: taskType, Runner_Status: status, Total: 0})
+				tasks = append(tasks, Task{Type: taskType, Runner_Status: status, Datasource: "null", Total: 0})
```

Comment:
```suggestion
				tasks = append(tasks, Task{Type: taskType, Runner_Status: status, Datasource: "null", Total: 0})
```
"null" de type string c'est marrant :joy: